.github/workflows/create-release.yml

```
@@ -82,7 +80,101 @@ jobs:
+    - name: Install build dependencies
+      if: steps.check-release.outputs.should_release == 'true'
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install build twine
+    
+    # NEW: Verify version matches tag
+    - name: Verify package version matches tag
+      if: steps.check-release.outputs.should_release == 'true'
+      run: |
+        TAG_VERSION="${{ steps.find-tags.outputs.version }}"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)